tests/src/test_app.py

```diff
@@ -427,7 +427,8 @@ def test_organization_hashtag_weekly_query():
             where name = 'msf'
             ),
             t2 as (
-                select name as hashtag, type as frequency , start_date , end_date , total_new_buildings , total_uq_contributors as total_unique_contributors , total_new_road_m as total_new_road_meters
```

Comment:
Update will be needed on Assertion Step ! Replacement of existing query with updated query that will be generated from query builder

Reply:
Fixed through https://github.com/hotosm/galaxy-api/pull/178/commits/3e963b3f63cd84b0f831b35bf97262c4547342cb and https://github.com/hotosm/galaxy-api/pull/178/commits/3c24122c9519a2891687b7f8db0b9e486208b6f4